index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>2048</title>
+  <title><span class="gurmukhi">2048</span></title>
```

Comment:
Unfortunately, this wouldn't work, as `<title>` only renders plain text. You would have to use unicode: ੨੦੪੮

Reply:
updated